src/main/java/org/ua/javaPro/berezhnoy/bank/hillelHomwWork28DBVersioning/Account.java

```diff
@@ -0,0 +1,26 @@
+package org.ua.javaPro.berezhnoy.bank.hillelHomwWork28DBVersioning;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name = "accounts")
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class Account {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
```

Comment:
```suggestion
    private Long id;
```
коли значення може бути пустим